twitter.py

```diff
@@ -1,59 +1,62 @@
 import base64
+import collections
 import json
 
-import requests
 from requests_oauthlib import OAuth1
 
 from environment_variables import get_env
+from request import request
+
+endpoint = collections.namedtuple('twitter_method', ["url", "type"])
```

Comment:
twitter_method -> endpoint (or endpoint -> twitter_method). Class name should preferably match the name in code.

Reply:
also fixed in flickr